index.html

```diff
@@ -3410,6 +3410,191 @@ <h3>
         <h2>
 DID URL Dereferencing
         </h2>
+        <p>
+The DID URL dereferencing function dereferences a <a>DID URL</a> into
+a resource with contents depending on the <a>DID URL</a>'s components,
+including the <a>DID method</a>, method-specific identifier, query parameters,
```

Comment:
```suggestion
including the <a>DID method</a>, method-specific identifier, path, query,
```
Propose to 1. include path, since that is also part of the DID URL, and 2. not assume here that the query always contains key=value parameters, even though that's what we mostly expect (see our discussion in https://github.com/w3c/did-core/issues/385).

Reply:
+1